changelog/812.feature

```diff
@@ -0,0 +1 @@
+Partially restore old initial batch distribution algorithm in ``LoadScheduling``.
```

Comment:
Can you please elaborate this? Something like what you used in the PR description would be great. 👍

Reply:
Done